src/puya/ir/builder/main.py

```diff
@@ -238,6 +238,98 @@ def visit_assignment_expression(self, expr: awst_nodes.AssignmentExpression) ->
         else:
             return ValueTuple(expr.source_location, list(result))
 
+    def visit_biguint_postfix_unary_operation(
+        self, expr: awst_nodes.BigUIntPostfixUnaryOperation
+    ) -> TExpression:
+        target_value = self.visit_and_materialise_single(expr.target)
+        rhs = BigUIntConstant(value=1, source_location=expr.source_location)
+        match expr.op:
```

Comment:
is it better to do this with a dictionary wrapped in `try ... except KeyError` ? I guess we lose compile time safety over the exhaustive check

Reply:
I prefer the way you've got it, given it's located in an entirely different layer (IR) to where it's defined (AWST)